modules/space/rendering/renderablestars.cpp

```diff
@@ -654,132 +574,99 @@ RenderableStars::RenderableStars(const ghoul::Dictionary& dictionary)
-    addProperty(_otherDataRange);
 
+    _otherDataColorMapPath.onChange([this]() { _otherDataColorMapIsDirty = true; });
+    if (p.otherDataColorMap.has_value()) {
+        _otherDataColorMapPath = p.otherDataColorMap->string();
+    }
     addProperty(_otherDataColorMapPath);
-    _otherDataColorMapPath.onChange([this]() {
-        if (std::filesystem::exists(_otherDataColorMapPath.value())) {
-            _otherDataColorMapIsDirty = true;
-        }
-        else {
-            LWARNING(std::format("File not found: {}", _otherDataColorMapPath.value()));
-        }
-    });
 
-    _staticFilterValue = p.staticFilter;
-    _staticFilterReplacementValue =
-        p.staticFilterReplacement.value_or(_staticFilterReplacementValue);
 
-    addProperty(_filterOutOfRange);
-
-    _renderingMethodOption.addOption(
-        RenderMethod::PointSpreadFunction,
-        "Point Spread Function Based"
-    );
-    _renderingMethodOption.addOption(RenderMethod::TextureBased, "Textured Based");
-    addProperty(_renderingMethodOption);
+    _otherDataRange.setViewOption(properties::Property::ViewOptions::MinMaxRange);
+    addProperty(_otherDataRange);
 
-    _renderingMethodOption = codegen::map<RenderMethod>(p.renderMethod);
 
-    _pointSpreadFunctionTexturePath = absPath(p.texture.string()).string();
-    _pointSpreadFunctionFile = std::make_unique<File>(
-        _pointSpreadFunctionTexturePath.value()
-    );
-    _pointSpreadFunctionTexturePath.onChange([this]() {
-        _pointSpreadFunctionTextureIsDirty = true;
-    });
-    _pointSpreadFunctionFile->setCallback([this]() {
-        _pointSpreadFunctionTextureIsDirty = true;
-    });
-    _userProvidedTextureOwner.addProperty(_pointSpreadFunctionTexturePath);
+    _fixedColor.setViewOption(properties::Property::ViewOptions::Color, true);
+    addProperty(_fixedColor);
 
-    _psfMethodOption.addOption(PsfMethodSpencer, "Spencer's Function");
-    _psfMethodOption.addOption(PsfMethodMoffat, "Moffat's Function");
-    _psfMethodOption = PsfMethodSpencer;
-    _psfMethodOption.onChange([this]() { renderPSFToTexture(); });
-    _parametersOwner.addProperty(_psfMethodOption);
 
-    _psfMultiplyOption.addOption(AppBrightness, "Use Star's Apparent Brightness");
-    _psfMultiplyOption.addOption(LumSize, "Use Star's Luminosity and Size");
-    _psfMultiplyOption.addOption(
-        LumSizeAppBrightness,
-        "Luminosity, Size, App Brightness"
-    );
-    _psfMultiplyOption.addOption(AbsMagnitude, "Absolute Magnitude");
-    _psfMultiplyOption.addOption(AppMagnitude, "Apparent Magnitude");
-    _psfMultiplyOption.addOption(DistanceModulus, "Distance Modulus");
+    addProperty(_filterOutOfRange);
 
 
-    if (p.sizeComposition.has_value()) {
-        _psfMultiplyOption =
-            static_cast<int>(codegen::map<SizeComposition>(*p.sizeComposition));
-    }
-    else {
-        _psfMultiplyOption = 5;
+    auto markTextureAsDirty = [this]() {_pointSpreadFunctionTextureIsDirty = true; };
```

Comment:
```suggestion
    auto markTextureAsDirty = [this]() {_pointSpreadFunctionTextureIsDirty = true; };
    
```
since it does not really belong with the halo texture only